packages/language-server/api.d.ts

```diff
@@ -4,6 +4,10 @@
  */
 export declare const SERVER_PATH: string;
 
+export type ServerInitializationOptions = {
```

Comment:
Could you add a comment that these params are passed via the connection?

Reply:
I'll add it where it's read